airbyte-integrations/connectors/source-surveycto/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,23 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "baseline_ig",
```

Comment:
this is dependent on the form id created on SurveyCTO right?  Can we make it clear here and provide instructions on the README so that other maintainers can replicate this form?